frontend/src/host_orchestrator/orchestrator/userartifacts.go

```diff
@@ -48,14 +49,18 @@ type UserArtifactChunk struct {
 // Abstraction for managing user artifacts for launching CVDs.
 type UserArtifactsManager interface {
 	UserArtifactsDirResolver
+	// Update the artifact given the checksum.
+	UpdateArtifact(checksum string, chunk UserArtifactChunk) error
+	// Update the artifact given the checksum.
```

Comment:
please update the comment about StatArtifact

Reply:
Done.